util/CMakeLists.txt

```diff
@@ -474,6 +474,9 @@ if(WIN32)
     target_link_libraries(crashpad_util PRIVATE user32 version winhttp)
     if(MSVC)
         target_compile_options(crashpad_util PRIVATE "/wd4201")
+        if(CMAKE_SIZEOF_VOID_P EQUAL 4)
```

Comment:
lol, this is how to detect 32 bit?

Reply:
looks like it.
This block is essentially copied from https://github.com/getsentry/sentry-native/blob/0e17d7c238d784d0283da582165f2d4f85bbb0a8/CMakeLists.txt#L288-L290 and I don’t claim to fully understand what it does and why :-D